source/iostreams.tex

```diff
@@ -430,8 +430,12 @@
 The objects are constructed and the associations are established at some
 time prior to or during the first time an object of class
 \tcode{ios_base::Init} is constructed, and in any case before the body
-of \tcode{main}\iref{basic.start.main} begins execution.\footnote{If it is possible for them to do so, implementations should
-initialize the objects earlier than required.}
+of \tcode{main}\iref{basic.start.main} begins execution.
+
+\recommended
+If it is possible for them to do so, implementations should
+initialize the objects earlier than required.
```

Comment:
I don't like having a *Recommended practice* in the middle of a paragraph. How about this structure:

> 3. The objects are constructed [...] begins execution. The objects are not destroyed during program execution.[Footnote]
> 4. *Recommended practice*: If it is possible for them do to so, implementations should initialize the objects earlier than required.
> 5. The results if including `<iostream>` [...] static storage duration.